src/test/checkqueue_tests.cpp

```diff
@@ -333,7 +333,8 @@ BOOST_AUTO_TEST_CASE(test_CheckQueue_FrozenCleanup)
         // would get called twice).
         vChecks[0].should_freeze = true;
         control.Add(vChecks);
-        control.Wait(); // Hangs here
+        bool waitResult = control.Wait(); // Hangs here
```

Comment:
14969: requires 14935

Reply:
#14935 is marked as `VM_Staged`, but I didn't find it among open PRs.
@vijaydasmp do you plan to publish it for review soon? I can rebase my merge on top of your fixes once those are checked in.

Reply:
@UdjinM6 I added bitcoin#14935 to the list of PRs backported by this merge.